semperfiwebdesign/all-in-one-seo-pack
# Don't include static pages twice

Issue #2126

## Proposed changes

Fixes an issue where the WooCommerce shop page is included twice in the sitemap.
Also bases the timestamp for the WooCommerce shop page on the last modified product.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

1. Install this PR.
2. Install WooCommerce, complete the initial wizard and make sure the Shop page has been created.
3. Add a WooCommerce Product.
4. Check the sitemap and make sure the WooCommerce shop page is no longer included twice.
5. Check that the shop page's timestamp is based on the last modified product.
